packages/styles/src/tokens/typography.ts

```diff
@@ -6,6 +6,7 @@ export const fontSizes = {
   xs: "10px",
   sm: "12px",
   base: "14px",
+  md: "14px",
```

Comment:
According to our typings, it is - because `HvButton` is using it in its `size` prop (maybe it shouldn't, but it is)

`lineHeights` doesn't have that issue

Reply:
I'll remove this, as this Button-Typography coupling is a bit awkward. This means we'll keep the TS issue for now (I'll cast it to `any`)